package exception;

public class TestException {

	public static void main(String[] args) {
		try {
			String [] name = new String[1];
			nomes [2] = "Teste";
			if (nomes[0].equals("Maria Sophia")
					System.out.println("Eh Maria ...");) {
				
			}
			
		} catch (Exception e) {
			// TODO: handle exception
		}

	}

}
